src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.model;
+
+import java.util.Objects;
+
+public class Book {
+    private Long id;
+    private String title;
+    private int price;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public void setPrice(int price) {
+        this.price = price;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Book book = (Book) o;
+        return price == book.price
```

Comment:
Update the equality check for `price` to use `BigDecimal.compareTo()` for accurate comparison.